README.md

```diff
@@ -79,6 +79,8 @@ colcon build
 
 ## Run
 
+The `TURTLEBOT3_MODEL` environment variable must be set when running your simulation. Valid values are `burger`, `waffle`, and `waffle_pi`.
```

Comment:
Explain that this is only for the simulation application and/or move this closer to line 102. 
I've also added a copy&paste to _Running Simulation Application_.